http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java

```diff
@@ -41,6 +45,14 @@ private FeignClients() {}
      * com.fasterxml.jackson.datatype.guava.GuavaModule} and {@link com.fasterxml.jackson.datatype.jdk7.Jdk7Module}.
      */
     public static FeignClientFactory standard() {
+        return standard(DEFAULT_TIMEOUT_OPTIONS);
+    }
+
+    /**
+     * Provides a {@link FeignClientFactory} with an {@link ObjectMapper} configured with {@link
+     * com.fasterxml.jackson.datatype.guava.GuavaModule} and {@link com.fasterxml.jackson.datatype.jdk7.Jdk7Module}.
+     */
+    public static FeignClientFactory standard(Request.Options timeoutOptions) {
```

Comment:
there's currently no good support for write timeout in feign. should
probably fix it there first.

On Thu, Feb 25, 2016 at 10:29 AM Jason Zhao <notifications@github.com>
wrote:

> In http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java
> <https://github.com/palantir/http-remoting/pull/69#discussion_r54140366>:
>
> > @@ -41,6 +45,14 @@ private FeignClients() {}
> >       * com.fasterxml.jackson.datatype.guava.GuavaModule} and {@link com.fasterxml.jackson.datatype.jdk7.Jdk7Module}.
> >       */
> >      public static FeignClientFactory standard() {
> > +        return standard(DEFAULT_TIMEOUT_OPTIONS);
> > +    }
> > +
> > +    /**
> > +     * Provides a {@link FeignClientFactory} with an {@link ObjectMapper} configured with {@link
> > +     * com.fasterxml.jackson.datatype.guava.GuavaModule} and {@link com.fasterxml.jackson.datatype.jdk7.Jdk7Module}.
> > +     */
> > +    public static FeignClientFactory standard(Request.Options timeoutOptions) {
>
> this only sets read and connection right, what if we want write timeout?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/http-remoting/pull/69/files#r54140366>.
>